script/Ciel/Day 5/05_01_CIEL05_3E.txt

```diff
@@ -458,7 +458,8 @@ The cold that assails her doesn't bottom out at -100℃.
 -- Page 603, Offset 12440.
 -- 局地的な低下はそれ以上はないとされるマイナス273℃にまで到達する。
 // Get this man a job in the superconductor industry
-The localized drop in temperatures has dipped all the way to -273℃, the absolute limit.
+// A little rounding is fine
```

Comment:
Submitter pointed out that _technically_ it's 273.15 but what's .15 degrees between friends.

Reply:
Source text says 273, no change necessary